src/libcore/cell.rs

```diff
@@ -880,6 +883,9 @@ impl<T:Default> Default for RefCell<T> {
 
 #[stable(feature = "rust1", since = "1.0.0")]
 impl<T: ?Sized + PartialEq> PartialEq for RefCell<T> {
+    /// # Panics
+    ///
+    /// Panics if the value is currently mutably borrowed.
```

Comment:
For the functions that take an `other` like this one, what do you think about using this wording:

> Panics if the value in either `RefCell` is currently borrowed.

This wording is taken from `RefCell::swap`

Reply:
Sounds like a great improvement 👍